app/views/application/_background.html.erb

```diff
@@ -0,0 +1,37 @@
+<%-
+  $background = "navbar-overlap"
+  $new_background = "navbar-overlap background"
+  $no_background = "navbar-overlap"
+  case (request.user_agent =~ /Mobile|webOS/).present? # check if user is using a mobile device
+    when true   # in case it is a mobile device
+      case controller_name
+        when ""
+          $background = $no_background
+        else
+          $background = $no_background
+      end
+    else  # in case it is not a mobile device
+      case controller_name
+        when "sessions"
+          case action_name
+            when "new"
+              $background = $new_background
+            else
+              $background = $no_background
+          end
+        when "users"
+          case action_name
+            when "new"
```

Comment:
Since the actions new and create do have the same behaviour, you can write something like `when "new", "create"` instead of having multiple "when".